src/day-1-toy/printf.py

```diff
@@ -6,6 +6,12 @@
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
 
+print('x is %s, y is %s, z is %s' % (x,y,z))
```

Comment:
This seems to give an output that is close to the desired output even though you've specified that x and y are string types with %s. It still outputs a number, but the more appropriate form of the solution is as follows: 
```
print('x is %d, y is %.2f, z is %s.' % (x, y, z))
```
Here f stands for a float number and the 0.2f makes it so that the float number outputs to two decimal places.